### DISCUSSION TITLE
How do I request a desired username that is already taken but does not have activity for over 4 years

### DISCUSSION THREAD

**Discussion by thaddavis** [community/community#48849] [New to GitHub]
How do I contact the account holder to see if I can have their username?

**Comment by queenofcorgis** [comment]
Hi @thaddavis,

We are no longer accepting requests to release dormant usernames. We are not able to accommodate this request.

If you hold a trademark to this name and believe there is a clear case of confusion between the account and your trademark, please review our [Trademark Policy](https://docs.github.com/articles/github-trademark-policy/) to determine if you should submit a trademark policy violation report.

Thank you!
